st2common/st2common/content/loader.py

```diff
@@ -204,6 +240,36 @@ def _get_folder(self, pack_dir, content_type):
             return None
         return path
 
+    def _get_runners_from_dir(self, base_dir):
+        result = {}
+        for runner_name in os.listdir(base_dir):
```

Comment:
I can't conceive of a time when this would be a big deal, but would be trivial to implement.